extensions/amp-jwplayer/1.0/base-element.js

```diff
@@ -76,6 +76,9 @@ BaseElement['props'] = {
   'adMacros': createParseAttrsWithPrefix('data-ad-macro-'),
   'config': createParseAttrsWithPrefix('data-config-'),
   'autoplay': {attr: 'autoplay', type: 'boolean'},
+  // TODO(wg-components): Current behavior defaults to loading="auto".
+  // Refactor to make loading="lazy" as the default.
```

Comment:
is this as easy as flipping the the default in `src/preact/base-element:PreactBaseElement.defaultProps_`?

Reply:
Nothing is easy 🙃  `loading="auto"` is baked into [AMP context](https://github.com/ampproject/amphtml/blob/main/src/preact/context.js#L29), so it's not trivial to change this.